### PR TITLE
Collect a list of obtained reward

### PR DESCRIPTION
**Pull request by mohsen-ghaffari1992**

### CODE REVIEW

**Comment by wasowski** [comment]
I started to reflect on this, and I wonder why do we need the list of lists of rewards. Don't we just need the final reward for each episode (wouldn't this be much cheaper)? Do we need to know the intermedia rewards during an episode? Do I miss something?

**Comment by mohsen-ghaffari1992** [comment]
> I started to reflect on this, and I wonder why do we need the list of lists of rewards. Don't we just need the final reward for each episode (wouldn't this be much cheaper)? Do we need to know the intermedia rewards during an episode? Do I miss something?

Of course, we need it. The policy is not only last action!

Imagine an example like the following (values are obtained reward in each step):
Episode 1: -1, -1, -1, 10
Episode 2: -1, -1, -1, -1, -1, -1, -1, -1, -1, 10

**Comment by mohsen-ghaffari1992** [comment]
@wasowski 
I pushed the new version that only stores the accumulated reward per episode.

File: src/test/scala/symsim/ExperimentSpec.scala [hunk 1]
```diff
@@ -16,7 +17,16 @@ trait ExperimentSpec[State, ObservableState, Action]
     outputToFile: Option[String] = None
   ): setup.Policy =
 
-    val q = setup.runQ
+    val (q, r) = setup.runQ
+//    val sums: List[Double] = r.map(_.sum)
```
Comment (wasowski) [inline src/test/scala/symsim/ExperimentSpec.scala:21]: This commented line should be removed.

File: src/test/scala/symsim/examples/concrete/braking/Experiments.scala [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Experiments
      alpha = 0.1,
      gamma = 0.1,
      epsilon = 0.05,
-     episodes = 100000,
+     episodes = 10000,
```
Comment (wasowski) [inline src/test/scala/symsim/examples/concrete/braking/Experiments.scala:16]: This change is unnecessary for this PR?

**Review by wasowski — COMMENTED** [review]
This is almost ready - some minor comments.

**Comment by mohsen-ghaffari1992** [comment]
@wasowski
Your comments are now applied. Should I merge it?

**Pull request merged**